### DISCUSSION TITLE
Is it compatible with Angular 12?

### DISCUSSION THREAD

**Discussion by VaskorCb** [themesberg/flowbite#815] [Q&A]
I just want to know, is it compatible with Angular 12? Please someone let me know.

**Comment by maxlshk** [comment]
The latest version of Flowbite for Angular is marked as unreleased for almost 2 years, according to its [npm package page](https://www.npmjs.com/package/flowbite-angular)​​. As far as I know the project might not be actively maintained or fully compatible even with the latest versions of Angular. So basically there is a [Flowbite repo](https://github.com/themesberg/flowbite-angular) for angular, but it's still in it's development stage and may not work well especially with older versions of the framework like 12-th release.

For more info you can refer to Flowbite-Angular official [documentation](https://flowbite.com/docs/getting-started/angular/).
